website/docs/r/private_ip_mode.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: private_ip_mode"
+sidebar_current: "docs-mongodbatlas-resource-private-ip-mode"
+description: |-
+    Provides a Private IP Mode resource.
+---
+
+# mongodbatlas_private_ip_mode
+
+`mongodbatlas_private_ip_mode` provides a Private IP Mode resource. This allows project to enable Connect via Peering Only mode.
+
+
```

Comment:
Add in box (so we call out here only applicable to GCP/Azure): 

What is Connect via Peering Only Mode?

Connect via Peering Only mode prevents clusters in an Atlas project from connecting to any network destination other than an Atlas Network Peer. **Connect via Peering Only mode applies only to GCP and Azure-backed dedicated clusters**. This setting disables the ability to:

- Deploy non-GCP or Azure-backed dedicated clusters in an Atlas project, and
- Use MongoDB Stitch with dedicated clusters in an Atlas project.